### PR TITLE
feat: 2.0.1

### PR DESCRIPTION
**Pull request by djschleen**

### CODE REVIEW

**Comment by djschleen** [comment]
closes #21

**Comment by djschleen** [comment]
closes #22

**Comment by djschleen** [comment]
closes #23

**Comment by djschleen** [comment]
closes #24

**Comment by djschleen** [comment]
closes #25 
closes #24 
closes #26

File: .hookz.yaml [hunk 1]
```diff
@@ -1,16 +1,25 @@
...
+        - name: "Go Build (Ensure pulled modules do not break the build)"
           exec: go
-          args: ["mod", "tidy"]
+          args: ["build"]
         - name: "Update all go dependencies to latest"
           exec: go
           args: ["get", "-u", "./..."]
+        - name: "Run gofmt to format the code"
+          exec: gofmt
+          args: ["-w", "*/*.go"]
```
Comment (juliojimenez) [inline .hookz.yaml:22]:
```suggestion
          args: ["-s", "-w", "**/*.go"]
```
Do you want to add simplification and recursive glob as a test?

**Review by juliojimenez — APPROVED** [review]

File: cmd/reset.go [hunk 2]
```diff
@@ -13,7 +13,8 @@ var (
 		Long:  "Rebuilds the hooks as defined in the .hooks.yaml file.",
 		Run: func(cmd *cobra.Command, args []string) {
 			hookzHeader()
-			fmt.Println("Resetting git hooks...")
+			fmt.Println("Resetting git hooks\n")
```
Comment (juliojimenez) [inline cmd/reset.go:16]:
```suggestion
			fmt.Println("Resetting git hooks")
			fmt.Println()
```
This seems to be causing a test failure.

**Pull request merged**